### ISSUE TITLE
Smarter Signature Removal

### ISSUE THREAD

**Issue by mrtoner** [robertklep/quotefixformac#45]
I finally figured out why QuoteFix sometimes won't leave quoted material in a reply: it removes it because it thinks the material is part of the signature.

![screenshot of mail 7-11-15 5 51 48 pm](https://cloud.githubusercontent.com/assets/1012776/8635899/8a78a72e-27f5-11e5-8a26-7f588c46d01d.png)

When "Don't remove sender's signature" is disabled, regardless of the "Remove quotes at this level and deeper" setting, QuoteFix removes everything after "Thank you".

If QuoteFix could be a little smarter in this regard, I'd be happy. :)

**Comment by alvarnell** [comment]
Not sure how QuoteFix determines things like that, but I know most e-mail client apps consider anything that follows two or more dashes in a row as the start of a signature. Any time I have the need to insert a line in a message I use equal signs

=================

Sent from Janet's iPad

-Al-
-- 
Al Varnell

**Comment by mrtoner** [comment]
Right, I assume that's what QF is doing. But it could leave alone any quoted material after/within the signature.

**Comment by alvarnell** [comment]
A fine example of why, when you top post, the signature should be at the bottom.

On Sat, Jul 11, 2015 at 07:21 PM, Don Morris wrote:
> Right, I assume that's what QF is doing. But it could leave alone any quoted material after/within the signature.
> 
> —
> Reply to this email directly or view it on GitHub.
> 

-Al-
-- 
Al Varnell
Mountain View, CA

**Comment by robertklep** [comment]
I have to agree with @alvarnell on this: when quoted text appears below a signature separator, those quotes become part of the signature (and hence get removed). 

That said, I'll have a look if it's easy to keep quoted text that is included after a signature separator. It may take some time, because priority number 1 is getting the plug-in to work on El Capitan :-)
